src/test/java/training/ItemsTest.java

```diff
@@ -0,0 +1,14 @@
+package training;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ItemsTest {
+
+    @Test
+    public void putItem() {
+
+        Items obj = new Items();

```

Comment:
This test does not provide any value obviously if you create an object it will not be null. What are you trying to test here?

Reply:
You did not correct this one.